tests/test_llm_runtime.py

```diff
@@ -28,28 +28,37 @@ def tearDownClass(cls) -> None:
 
     def test_llm_runtime(self):
         model_name = "/tf_dataset2/models/nlp_toolkit/llama-2-7b-chat/Llama-2-7b-chat-hf"
-        woq_config = WeightOnlyQuantConfig(compute_dtype="int8", weight_dtype="int4", use_cache=True, not_quant=True)
         prompt = "What is the meaning of life?"
 
         tokenizer = AutoTokenizer.from_pretrained(model_name, trust_remote_code=True)
         inputs = tokenizer(prompt, return_tensors="pt")
-
-        # pytorch fp32
-        pt_model = AutoModelForCausalLM.from_pretrained(model_name, trust_remote_code=True)
-        pt_model.eval() 
-        pt_logits = pt_model(input_ids=inputs.input_ids).logits[:,-1]
-        pt_generate_ids = pt_model.generate(input_ids=inputs.input_ids, do_sample=False, max_new_tokens=100)[0].tolist()
+        
+        pt_logits = torch.load("/tf_dataset2/inc-ut/nlptoolkit_ut_model/llama2_pt_logits.pth")[:,-1]
```

Comment:
if this is just for CI, shall we designate the path under .github/

Reply:
This is the path inside docker container

Reply:
can we setup an environmental variable for path since users might want to verify these UT?

Reply:
Users still need to prepare those local file to run UTs no matter setup or not setup a variable
Actually the best way is to judge if you have local file then use local file, if you don't have local file, UT can lead you to generate those. In this way many of our UT need to rewrite.